elements/pfe-primary-detail/src/pfe-primary-detail.js

```diff
@@ -300,16 +339,134 @@ class PfePrimaryDetail extends PFElement {
         details: nextDetails
       }
     });
+  } // end _handleHideShow()
 
-    // Set focus to pane
-    const firstFocusableElement = nextDetails.querySelector(
-      'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])'
-    );
-    if (firstFocusableElement) {
-      firstFocusableElement.focus();
+  /**
+   * A11y features
+   */
+  _isToggle(element) {
+    return element.classList.contains("a11y-toggle");
+  }
+
+  _isPanel(element) {
+    return element.classList.contains("a11y-panel");
```

Comment:
Could do `return element.getAttribute('slot') === 'details'`?

Reply:
@wesruv will that return the tab panels themselves or the wrapper for the panels?

Reply:
That'll return true || false depending on the HTML attribute slot being set to `details` (or not)

Reply:
I'm not sure that I understand the question?

If the concern is you might be passing a child element of the slotted tag you could do:
`return element.getAttribute('slot') === 'details' || element.parentElement.getAttribute('slot') === 'details'`

Reply:
@wesruv I just misunderstood before but I understand now. Thank you!